package.json

```diff
@@ -16,7 +16,10 @@
     "antd": "^5.17.2",
     "axios": "^1.7.2",
     "bootstrap": "^5.3.3",
+    "docx": "^8.5.0",
+    "file-saver": "^2.0.5",
     "http": "^0.0.1-security",
+    "jspdf": "^2.5.1",
```

Comment:
Why do we need these packages? remove if not needed

Reply:
Done!